parser/parser.js

```diff
@@ -296,11 +301,15 @@ export const parseParagraph = (documentContext) => (paragraph) => {
       md.join("").replaceAll("\n", "\n" + leadingSpace + "    ")
     );
   } else {
+    // Add quote marker if the paragraph is indented beyond our threshold
+    const isQuote = indentStart >= QUOTE_INDENT_THRESHOLD;
+    const quotePrefix = isQuote ? "> " : "";
```

Comment:
A bit of a nit so no problem if you want to keep it as is, but a consideration could be that this is introducing a nested conditional (the ternary in the `else`). If text classifications are mutually exclusive (i.e. the text can either be bullet, block quote or normal) then having the conditionals in a single level chain can make this clearer, IMO.

To lift the conditional, the `isQuote` logic could be moved next to where `indentStart` and `QUOTE_INDENT_THRESHOLD` are declared (might be good to have them all together anyways) or a small `isQuote(paragraphStyle)` function could be defined.
```
// Add quote marker if the paragraph is indented beyond our threshold
const isQuote = indentStart >= QUOTE_INDENT_THRESHOLD;
```

Then the conditional would just be
```suggestion
  } else if (isQuote) {
    const quotePrefix = "> ";
```

Note an `else` statement with a separate return would still be necessary for normal text.

Reply:
I ended up defining quotePrefix everywhere as an empty string, and updating it if needed. LMK what you think of this type of logic

Reply:
Yea, current logic looks good to me 👍